### DISCUSSION TITLE
MMKV是否可以实现存储在不同的文件中,增加存储和读取速度。

### DISCUSSION THREAD

**Discussion by 2544921714** [Tencent/MMKV#1055] [Q&A]
MMKV是否可以实现存储在不同的文件中,增加存储和读取速度。

**Comment by lingol** [comment]
You should separate different sets of key-value base on your needs (such as business logic, module, whatever you call it) to different MMKV instances.

**Reply by 2544921714** [reply]
final dir = await getApplicationSupportDirectory();
  var mmkv = MMKV('testCustomDir', rootDir: dir.path + '/mmkv_3');
像这样吗?定义多个实例,然后定义不同存储位置吗?

**Comment by lingol** [comment]
You don't need a customize root directory. A different MMKV ID will do the job.

**Reply by lingol** [reply]
https://github.com/Tencent/MMKV/wiki/iOS_tutorial_cn#crud-%E6%93%8D%E4%BD%9C

**Reply by 2544921714** [reply]
var mmkv = MMKV("test");
  mmkv.encodeBool('bool', true);
这样可以自定义不同的实例,然后根目录是同一个,但是根目录中是可以区分不同的实例,那我们直接获取的时候不同的实例就可以了,是这样吗?

**Reply by lingol** [reply]
Right.

**Reply by 2544921714** [reply]
flutter中获取实例的具体方法,具体是什么呢?文档中我没看见具体体现呢...

**Reply by lingol** [reply]
WHAT？https://github.com/Tencent/MMKV/wiki/flutter_setup

**Reply by 2544921714** [reply]
var mmkv = MMKV("test");
mmkv.encodeBool('bool', true);这个是创建实例,那获取某一实例内容是直接get("test")?

**Reply by lingol** [reply]
WHAT？？？
